Userland/Applications/VideoPlayer/VideoPlayerWidget.cpp

```diff
@@ -87,21 +88,30 @@ ErrorOr<void> VideoPlayerWidget::setup_interface()
     find_descendant_of_type_named<GUI::Button>("fullscreen")->set_action(*m_toggle_fullscreen_action);
 
     m_size_fit_action = GUI::Action::create_checkable("&Fit", [&](auto&) {
-        m_video_display->set_sizing_mode(VideoSizingMode::Fit);
+        set_sizing_mode(VideoSizingMode::Fit);
     });
 
     m_size_fill_action = GUI::Action::create_checkable("Fi&ll", [&](auto&) {
-        m_video_display->set_sizing_mode(VideoSizingMode::Fill);
+        set_sizing_mode(VideoSizingMode::Fill);
     });
 
     m_size_stretch_action = GUI::Action::create_checkable("&Stretch", [&](auto&) {
-        m_video_display->set_sizing_mode(VideoSizingMode::Stretch);
+        set_sizing_mode(VideoSizingMode::Stretch);
     });
 
     m_size_fullsize_action = GUI::Action::create_checkable("F&ull Size", [&](auto&) {
-        m_video_display->set_sizing_mode(VideoSizingMode::FullSize);
+        set_sizing_mode(VideoSizingMode::FullSize);
     });
 
+    // Load the current video sizing mode
+    // The default fallback for `read_u32` is 0, which is also our desired default for the sizing mode, VideoSizingMode::Fit
+    auto sizing_mode_value = Config::read_u32("VideoPlayer"sv, "Playback"sv, "SizingMode"sv);
+    if (sizing_mode_value >= to_underlying(VideoSizingMode::Sentinel)) {
+        sizing_mode_value = 0;
```

Comment:
Perhaps in the future this could complain "SizingMode value {} is out of bounds, default to Fit". And in the future future we could have helpers to `read_enum()` or `read_bounded<u32>(sv, sv, sv, min, max, fallback)` or some such.